### PR TITLE
Inactive input fields

### PR DESCRIPTION
**Pull request by davidimmhahn**
Shows sliders and radio buttons as inactive until first user input for respective feature is given.

### CODE REVIEW

File: lib/widgets/radioButtons.dart [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import 'package:flutter/material.dart';
 import 'package:bars_frontend/main.dart';
```
Comment (davidimmhahn) [inline lib/widgets/radioButtons.dart]:
```suggestion
```

File: lib/widgets/sliders.dart [hunk 2]
```diff
@@ -1,31 +1,74 @@
 import 'package:flutter/material.dart';
 import 'package:bars_frontend/main.dart';
```
Comment (davidimmhahn) [inline lib/widgets/sliders.dart]:
```suggestion
```

File: lib/widgets/sliders.dart [hunk 3]
```diff
@@ -1,31 +1,62 @@
...
-          },
+Widget getSliderInputRow(MyHomePageState homePageState, State context,
+    MapEntry<String, dynamic> feature) {
+  return SliderInputRow(homePageState, context, feature);
+}
+
+/// Defines a slider that changes its color when dragged and is correlated to [feature].
+class SliderInputRow extends StatelessWidget {
+  final MyHomePageState homePageState;
+  final State sliderContext;
```
Comment (justus-hildebrand) [inline lib/widgets/sliders.dart]: sliderContext is a State so call it sliderState, no?

Comment (justus-hildebrand) [reply lib/widgets/sliders.dart]: rename to `parentState`

**Review by justus-hildebrand — APPROVED** [review]

**Pull request merged**